src/daemon/http/server.rs

```diff
@@ -531,11 +531,16 @@ pub async fn metrics(req: Request) -> RoutingResult {
                     }
                 }
 
-                if !server.config.metrics.metrics_hide_child_details {
+                // Do not show child metrics if none of the CAs has any children..
+                // Many users do not delegate so, showing these metrics would just be confusing.
+                let any_children = cas_stats.values().any(|ca| ca.child_count() > 0);
```

Comment:
A minor thing, but would `!is_empty()` here be more idiomatic and potentially less computation?

Reply:
no, child_count is just a number on `CertAuthStats`.